src/teleop/jetson/jetson_teleop.py

```diff
@@ -133,6 +143,39 @@ def ra_mode_callback(self, msg: String) -> None:
         with self._arm_mode_lock:
             self._arm_mode = msg.data
 
+    def brushless_encoder_callback(self, msg: MotorsStatus) -> None:
+        """
+        Callback for brushless encoder values
+        :param msg: MotorsStatus message representing the current status of the brushless motors
+        :return:
+        """
+        with self._joint_state_lock:
+            for i, name in enumerate(msg.name):
+                index_of_joint = self.current_ra_joint_states.name.index(name)
+                self.current_ra_joint_states.position[index_of_joint] = msg.joint_states.position[i]
+                self.current_ra_joint_states.velocity[index_of_joint] = msg.joint_states.velocity[i]
+                self.current_ra_joint_states.effort[index_of_joint] = msg.joint_states.effort[i]
+
+    def brushed_encoder_callback(self, msg: JointState) -> None:
+        """
+        Callback for brushed encoder values
+        :param msg: JointState message representing the current joint states of the brushed motors
+        :return:
+        """
+        with self._joint_state_lock:
+            for i, name in enumerate(msg.name):
+                index_of_joint = self.current_ra_joint_states.name.index(name)
```

Comment:
Do you guys actually use effort? If so, the brushed_encoder effort will actually be garbage.

Reply:
No, I just added this for cohesiveness I guess, probably not neccesary